src/BBT.StructureTools/StructureToolsException.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace BBT.StructureTools
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// A general exception indicating that an unrecoverable error
+    /// occured within the StructureTools library.
+    /// </summary>
+    [Serializable]
+    public class StructureToolsException : Exception
```

Comment:
Good naming, please clean up the usages of the [CopyConvertCompareException](https://github.com/bbtsoftware/BBT.StructureTools/blob/develop/src/BBT.StructureTools/CopyConvertCompareException.cs) so we don't have two excpetion types for the same reason.

Reply:
done.